benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -7,6 +7,83 @@
 
 import React from "react";
 
+const getFilterableData = ({ tools, rows }) => {
```

Comment:
I guess due to the amount of code it is now the time to move all filter-specific utils to a separate file like `utils/filters.js`. General methods like `omit` can stay here.

Could you also please add a short (JavaDoc-style) comment on each utility method that you add?